routes/baseDBMethods.js

```diff
@@ -20,15 +22,48 @@ function handleDocResponse(ctx, doc) {
   }
 }
 
+function processPaginationParameters(ctx) {
+  let pageNumber = parseInt(ctx.query.page, 10);
+  if (Number.isNaN(pageNumber)) {
+    pageNumber = DEFAULT_PAGE_NUMBER;
+  } else if (pageNumber < 1) {
+    throw new RangeError('Query parameter \'page\' must be a positive number');
+  }
+
+  let pageSize = parseInt(ctx.query.page_size, 10);
+  if (Number.isNaN(pageSize)) {
```

Comment:
Try to use: let pageSize = parseInt(ctx.query.page_size, 10) || DEFAULT_PAGE_SIZE
I am curious if it will work, if it will, we'll be avoiding ugly if :)

Reply:
NICE! It works!